Code/index.js

```diff
@@ -0,0 +1,2 @@
+export Block from './Block'
```

Comment:
This is actually related to having a built version: with a built version, this is a non-issue. Let's follow up on https://github.com/klarna/ui/issues/81